addons/amxmodx/scripting/rt_bonus.sma

```diff
@@ -76,14 +76,15 @@ public rt_revive_end(const iEnt, const id, const activator, const modes_struct:m
 		if(g_eCvars[HEALTH])
 		{
 			set_entvar(id, var_health, g_eCvars[HEALTH]);
+            set_entvar(id, var_max_health, g_eCvars[HEALTH]);
```

Comment:
var_max_health - has no place, because on some servers the limit may be HIGHER than the maximum possible health when giving a bonus.

It is necessary to remove from the PR

Reply:
the user should not get more health than сvar, this basically only applies to :TakeHealth() which is controlled by pev->max_health